Source/UnitTests/Core/IOS/FS/FileSystemTest.cpp

```diff
@@ -434,3 +434,21 @@ TEST_F(FileSystemTest, ReadDirectoryOrdering)
   ASSERT_EQ(result->size(), file_names.size());
   EXPECT_TRUE(std::equal(result->begin(), result->end(), file_names.rbegin()));
 }
+
+TEST_F(FileSystemTest, CreateFullPath)
```

Comment:
Should any of these pathes be deleted after the test is finished?  Otherwise the next time it runs the test won't be operating on a clean slate

Reply:
Yeah, they should. This is automatically done by the testing framework since we're using a fixture. Each test that is defined with TEST_F uses its own instance of FileSystemTest and (by extension) its own file system root, so the tests are independent.